proto/destination.proto

```diff
@@ -101,6 +101,14 @@ message WeightedAddr {
   map<string, string> metric_labels = 4;
   TlsIdentity tls_identity = 5;
   ProtocolHint protocol_hint = 6;
+  RemoteServiceData remote_service_data = 7;
+}
+
+// A collection of metadata, useful for
+// communicating with services that are
+// outside the scope of the local cluster
+message RemoteServiceData {
+    string fully_qualified_name = 1;
```

Comment:
We should be specific about if this is a FQDN or an authority. (the difference being that an authority contains a port)